frontend/src/components/layouts/AppLayout.tsx

```diff
@@ -22,6 +23,65 @@ function AppLayoutInner() {
   useRemoveSuccessfulChannelOrder();
   useNotifyReceivedPayments();
 
+  // Update document.title and the history entry when the route changes.
```

Comment:
Could this be moved to a separate hook? see the ones two lines above for examples